package.json

```diff
@@ -33,7 +33,9 @@
     "next": "14.0.4",
     "react": "^18",
     "react-dom": "^18",
+    "react-loader-spinner": "^6.1.6",
     "react-telegram-login": "^1.1.2",
+    "reaviz": "^15.18.3",
```

Comment:
The other charts rendered in the forge, in this repository, are using `recharts`, because of how good it is at customization

For the sake of not having 2 libraries for the same purpose, please remove his library and Migrate the use of the `PieCharts` to `recharts`. 

Here's an example that might be useful, to accomplish the graphic needed in this feature:

```
import React, { PureComponent } from 'react';
import { PieChart, Pie, Legend, Cell, ResponsiveContainer } from 'recharts';

const data01 = [
  { name: 'Group A', value: 400, fill: '#0088FE' },
  { name: 'Group B', value: 300, fill: '#00C49F' },
  { name: 'Group C', value: 300, fill: '#FFBB28' },
  { name: 'Group D', value: 200, fill: '#FF8042' },
];

export default class Example extends PureComponent {
  static demoUrl = 'https://codesandbox.io/s/two-simple-pie-chart-otx9h';

  render() {
    return (
      <ResponsiveContainer width="100%" height="100%">
        <PieChart width={400} height={400}>
          <Pie
            dataKey="value"
            data={data01}
            cx="50%"
            cy="50%"
            outerRadius={80}
            fill="#FFF"
            label
          />
        </PieChart>
      </ResponsiveContainer>
    );
  }
}
```

There's a sample in their official page, with the possibility to modify the chart there to preview changes before integrating, that's where I got this example from.

https://recharts.org/en-US/examples/TwoSimplePieChart

PS: You can bring changes from `dev` or `main` with no issues. Currently, `main` and `dev` branches have the same changes